packages/@okta/i18n/src/properties/login_da.properties

```diff
@@ -105,7 +105,7 @@ errors.E0000026 = Dette slutpunkt er for\u00E6ldet.
 errors.E0000027 = Gruppe-push ugyldig anmodning : {0}
 errors.E0000028 = Anmodningen mangler en p\u00E5kr\u00E6vet parameter.
 errors.E0000029 = Ugyldig persons\u00F8gning.
-errors.E0000030 = Ugyldig anmodning. Ugyldig dato. Datoer skal v\u00E6re i formatet \u00C5\u00C5\u00C5\u00C5-MM-dd''T''TT:mm:ss.SSSZZ, f.eks. 2013-01-01T12:00:00.000-07:00.
+errors.E0000030 = Ugyldig anmodning. Ugyldig dato. Datoer skal v\u00E6re i formatet YYYY-MM-dd''T''HH:mm:ss.SSSZZ, f.eks. 2013-01-01T12:00:00.000-07:00.
```

Comment:
Is this XTM fixing a date format that they previously translated ?

Reply:
Yes 😆 . They're actually fixing a problem.